lib/util/constants.js

```diff
@@ -71,3 +71,6 @@ module.exports.typeErrors = {
 module.exports.DEFAULT_NQ_LEVEL_LOCAL = 1;
 module.exports.DEFAULT_NQ_LEVEL_REMOTE = 0;
 module.exports.MAX_NQ_LEVEL = 3;
+
+module.exports.ICE_MONITOR_ACTIVITY_CHECK_PERIOD_MS = 1000;
+module.exports.ICE_MONITOR_INACTIVITY_THRESHOLD_MS = 3000;
```

Comment:
@makarandp0 Nitpick: I think we can shorten this to `ICE_INACTIVITY_THRESHOLD_MS`.